tests/providers/test_braket_backend.py

```diff
@@ -319,3 +454,56 @@ def test_target(self):
         self.assertEqual(len(target.operations), 2)
         self.assertEqual(len(target.instructions), 60)
         self.assertIn("Target for Amazon Braket QPU", target.description)
+
+    def test_target_invalid_device(self):
+        """Tests target."""
+        mock_device = Mock()
+        mock_device.properties = None
+
+        with self.assertRaises(exception.QiskitBraketException):
+            aws_device_to_target(mock_device)
+
+    def test_fully_connected(self):
+        """Tests if instruction_props is correctly populated for fully connected topology."""
+        mock_device = Mock()
+        mock_device.properties = RIGETTI_MOCK_GATE_MODEL_QPU_CAPABILITIES.copy(
+            deep=True
+        )
+        mock_device.properties.paradigm.connectivity.fullyConnected = True
+        mock_device.properties.paradigm.qubitCount = 2
+        mock_device.properties.action.get(
+            DeviceActionType.OPENQASM
+        ).supportedOperations = ["CNOT"]
+
+        instruction_props = aws_device_to_target(mock_device)
+
+        from qiskit.circuit import Instruction
```

Comment:
Is there a special reason that the import is here? If not, I suggest to move the import to the top of this file.

Reply:
If it is to avoid confusion with Braket's Instruction, we can do something like `rom qiskit.circuit import Instruction as QiskitInstruction`